tests/src/test/java/alluxio/client/FileInStreamIntegrationTest.java

```diff
@@ -401,4 +401,20 @@ public void remoteReadLargeFile() throws Exception {
       }
     }
   }
+
+  @Test
+  @LocalAlluxioClusterResource.Config(
+      confParams = {PropertyKey.Name.USER_FILE_CACHE_PARTIALLY_READ_BLOCK, "false"})
+  public void positionedReadWithoutPartialCaching() throws Exception {
```

Comment:
What happens to this test before the fix?

Reply:
It throws exception of accessing position outside the file boundary.